shuup/admin/templates/shuup/admin/orders/_order_shipments.jinja

```diff
@@ -27,3 +35,26 @@
         </div>
     {% endif %}
 </div>
+
+{% block extra_js %}
+<script>
+    function handleDelete(event, link) {
+        event.preventDefault();
+        var warningMessage = "{% trans %}Shipped shipments should not be deleted. Delete will effect on products physical stock count. Delete anyway?{% endtrans %}";
```

Comment:
Second sentence should probably say "Deletion will affect products physical stock count"